### PR TITLE
Confirm edit

### PR DESCRIPTION
**Pull request by tkthib2015012**
Issue: #22

### CODE REVIEW

**Review by miljkovicjovan — CHANGES_REQUESTED** [review]
1. Change the text to "Are you sure you want to edit this task?<br>You will lose your previous one!"

2. When someone cancels the editMode should be false and get back to regular view.

Other than that works fine just delete comments.

**Comment by tkthib2015012** [comment]
> 1. Change the text to "Are you sure you want to edit this task?You will lose your previous one!"
> 2. When someone cancels the editMode should be false and get back to regular view.
> 
> Other than that works fine just delete comments.

okay, I will fix it

**Comment by tkthib2015012** [comment]
@miljkovicjovan I have updated it already

File: package-lock.json
Comment (miljkovicjovan) [inline package-lock.json:1]: Why has this file been changed ?

Comment (tkthib2015012) [reply package-lock.json:1]: I run the command npm install and it change.

**Review by miljkovicjovan — APPROVED** [review]
All good did some code cleanup and merging now thank you 😃

**Pull request merged**